creator-node/src/components/replicaSet/syncQueueComponentService.js

```diff
@@ -0,0 +1,13 @@
+
+/**
+ * Enqueues sync operation into syncQueueService for provided walletPublicKeys against provided creatorNodeEndpoint
+ */
+const enqueueSync = async ({ serviceRegistry, walletPublicKeys, creatorNodeEndpoint }) => {
+  await serviceRegistry.syncQueueService.enqueueSync(
```

Comment:
shouldn't we be able to directly call `syncQueueService.enqueueSync` here?

Reply:
unclear on this comment too

Reply:
do you mean i should pass in the syncQueueService directly instead of passing in the serviceRegistry?

Reply:
can we import syncQueueService and use it?

Reply:
or is there some timing issue or its null

Reply:
i don't see why that's better, generally the direct imports seem to cause us more pain, vs this component is only consumed by the service, so it seems better to pass it this way? I could replace the serviceRegistry param with the syncQueueService param and pull the field in the caller instead

Reply:
just seemed weird to call through service registry was all. i know it's initialized there but doesn't seem like it needs to be called from there but this is minor

Reply:
feel free to resolve if you think this pattern is okay

Reply:
resolved offline, leaving as is for now